### PR TITLE
part_06 exercises complete

### PR DESCRIPTION
**Pull request by mikram5**

### CODE REVIEW

File: src/part_06/ex_06/Exercise_06.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+class Class2 extends Super {
+
+    void method1() {
+        System.out.println("method1() in Class2");
+    }
+}
+
+class Class3 extends Class2 {
+
+    void method1() {
```
Comment (roioteromorales) [inline src/part_06/ex_06/Exercise_06.java]: Would be good to user the Annotation @Override  when you are overriding a method from a super class.

File: src/part_06/ex_06/Exercise_06.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+class Super {
+
+    void method1() {
+        System.out.println("method1() in Super");
+    }
+}
+
+class Class2 extends Super {
+
+    void method1() {
```
Comment (roioteromorales) [inline src/part_06/ex_06/Exercise_06.java]: Here you are Overriding too!

File: src/part_06/ex_06/Exercise_06.java (hunk 1 above)
Comment (mikram5) [reply src/part_06/ex_06/Exercise_06.java]: I made the requested change.

File: src/part_06/ex_06/Exercise_06.java (hunk 2 above)
Comment (mikram5) [reply src/part_06/ex_06/Exercise_06.java]: I made the requested change.

**Pull request merged**